controller/konnect/ops/ops_controlplane_test.go

```diff
@@ -408,92 +408,92 @@ func TestUpdateControlPlane(t *testing.T) {
+		// 		sdk := sdkmocks.NewMockControlPlaneSDK(t)
+		// 		sdkGroups := sdkmocks.NewMockControlPlaneGroupSDK(t)
+		// 		cp := &konnectv1alpha1.KonnectGatewayControlPlane{
+		// 			Spec: konnectv1alpha1.KonnectGatewayControlPlaneSpec{
+		// 				CreateControlPlaneRequest: sdkkonnectcomp.CreateControlPlaneRequest{
+		// 					Name: "cp-1",
+		// 				},
+		// 			},
+		// 			Status: konnectv1alpha1.KonnectGatewayControlPlaneStatus{
+		// 				KonnectEntityStatus: konnectv1alpha1.KonnectEntityStatus{
+		// 					ID: "12345",
+		// 				},
+		// 			},
+		// 		}
+		// 		sdk.
+		// 			EXPECT().
+		// 			UpdateControlPlane(ctx, "12345",
+		// 				sdkkonnectcomp.UpdateControlPlaneRequest{
+		// 					Name:        sdkkonnectgo.String(cp.Spec.Name),
+		// 					Description: cp.Spec.Description,
+		// 					AuthType:    (*sdkkonnectcomp.UpdateControlPlaneRequestAuthType)(cp.Spec.AuthType),
+		// 					ProxyUrls:   cp.Spec.ProxyUrls,
+		// 					Labels:      WithKubernetesMetadataLabels(cp, cp.Spec.Labels),
+		// 				},
+		// 			).
+		// 			Return(
+		// 				&sdkkonnectops.UpdateControlPlaneResponse{
+		// 					ControlPlane: &sdkkonnectcomp.ControlPlane{
+		// 						ID: lo.ToPtr("12345"),
+		// 					},
+		// 				},
+		// 				nil,
+		// 			)
+
+		// 		return sdk, sdkGroups, cp
+		// 	},
+		// 	expectedID: "12345",
+		// },
+		// {
+		// 	name: "fail",
+		// 	mockCPTuple: func(t *testing.T) (*sdkmocks.MockControlPlaneSDK, *sdkmocks.MockControlPlaneGroupSDK, *konnectv1alpha1.KonnectGatewayControlPlane) {
+		// 		sdk := sdkmocks.NewMockControlPlaneSDK(t)
+		// 		sdkGroups := sdkmocks.NewMockControlPlaneGroupSDK(t)
+		// 		cp := &konnectv1alpha1.KonnectGatewayControlPlane{
+		// 			ObjectMeta: metav1.ObjectMeta{
+		// 				Name:      "cp-1",
+		// 				Namespace: "default",
+		// 			},
+		// 			Spec: konnectv1alpha1.KonnectGatewayControlPlaneSpec{
+		// 				CreateControlPlaneRequest: sdkkonnectcomp.CreateControlPlaneRequest{
+		// 					Name: "cp-1",
+		// 				},
+		// 			},
+		// 			Status: konnectv1alpha1.KonnectGatewayControlPlaneStatus{
+		// 				KonnectEntityStatus: konnectv1alpha1.KonnectEntityStatus{
+		// 					ID: "12345",
+		// 				},
+		// 			},
+		// 		}
+
+		// 		sdk.
+		// 			EXPECT().
+		// 			UpdateControlPlane(ctx, "12345",
+		// 				sdkkonnectcomp.UpdateControlPlaneRequest{
+		// 					Name:        sdkkonnectgo.String(cp.Spec.Name),
+		// 					Description: cp.Spec.Description,
+		// 					AuthType:    (*sdkkonnectcomp.UpdateControlPlaneRequestAuthType)(cp.Spec.AuthType),
+		// 					ProxyUrls:   cp.Spec.ProxyUrls,
+		// 					Labels:      WithKubernetesMetadataLabels(cp, cp.Spec.Labels),
+		// 				},
+		// 			).
+		// 			Return(
+		// 				nil,
+		// 				&sdkkonnecterrs.BadRequestError{
+		// 					Status: 400,
+		// 					Detail: "bad request",
+		// 				},
+		// 			)
+
+		// 		return sdk, sdkGroups, cp
+		// 	},
+		// 	expectedErr: true,
+		// },
```

Comment:
?

Reply:
🤦 

a1f22eb1cdd67e3a8f2a880b772bee92eb06cca6 fixes that.